docs/quickstart.md

```diff
@@ -0,0 +1,209 @@
+# Quickstart
+
+In this guided tour, we'll cover how to write, verify, and run WDL documents
+using Sprocket. This should give you a good sense of where and how you might use
+Sprocket in your day-to-day work.
+
+To follow along, you are encouraged to download [this WDL
+document](./quickstart/example.wdl){target="_self" download="example.wdl"} that
+has been specifically crafted for this walkthrough.
+
+## Ensuring high-quality code
+
+Using automation to ensure high-quality code both during development and through
+continuous integration (CI) processes is one of the hallmarks of modern
+development. Sprocket contains both the `check` and `lint` (shortcut for `check
+--lint`) subcommands to help you during your WDL development.
```

Comment:
Should we link the `sprocket-action` repo here? Which I don't think ever got updated to the latest release, so we need to keep that in mind moving forward.

Reply:
I've added details on this.